### PR TITLE
updated to use torchvision =< 0.16.0

### PR DESCRIPTION
**Pull request by cuevas1208**
code updates to use torchvision version  =< 0.16.0

### CODE REVIEW

**Comment by Floyd-bit** [comment]
Hello, I am trying to run your code with HRSC2016-MS dataset and trained 100 epoches using RTRDETR/rtdetr_pytorch/configs/rtdetr/rtdetr_r18vd_6x_coco.yml configuration file but the AP metrics stayed at 0 during the training process.I would appreciate if you have valuable suggestions for this issue.
image
image

**Comment by Floyd-bit** [comment]
@aaronrmm @cuevas1208

File: rtdetr_pytorch/configs/rtdetr/include/dataloader.yml [hunk 1]
```diff
@@ -8,15 +8,15 @@ train_dataloader:
       ops:
         - {type: RandomPhotometricDistort, p: 0.5}
         - {type: RandomZoomOut, fill: 0}
-        - {type: RandomIoUCrop, p: 0.8}
```
Comment (aaronrmm) [inline rtdetr_pytorch/configs/rtdetr/include/dataloader.yml:11]: - [ ] change back to p=0.8 ?

File: rtdetr_pytorch/src/zoo/rtdetr/rtdetr_decoder.py [hunk 2]
```diff
@@ -698,9 +698,9 @@ def forward(self, feats, targets=None):
             dn_out_logits, out_logits = torch.split(
                 out_logits, dn_meta["dn_num_split"], dim=2
             )
-            dn_out_angles, out_angles = torch.split(
-                out_angles, dn_meta["dn_num_split"], dim=2
-            )
+            # dn_out_angles, out_angles = torch.split(
+            #     out_angles, dn_meta["dn_num_split"], dim=2
+            # )
```
Comment (aaronrmm) [inline rtdetr_pytorch/src/zoo/rtdetr/rtdetr_decoder.py]: - [ ] remove comments

File: rtdetr_pytorch/src/zoo/rtdetr/rtdetr_postprocessor.py [hunk 3]
```diff
@@ -58,7 +58,7 @@ def forward(self, outputs, orig_target_sizes):
             boxes = bbox_pred.gather(
                 dim=1, index=index.unsqueeze(-1).repeat(1, 1, bbox_pred.shape[-1])
             )
-            angles = angles.gather(dim=1, index=index)
+            # angles = angles.gather(dim=1, index=index)
```
Comment (aaronrmm) [inline rtdetr_pytorch/src/zoo/rtdetr/rtdetr_postprocessor.py]: - [ ] remove comments

File: rtdetr_pytorch/tests/unit/configs/test_rtdetr_r18vd_6x_coco_config.py [hunk 4]
```diff
@@ -146,7 +146,7 @@ def test_rtdetr_r18vd_6x_coco_yml():
                     "ops": [
                         {"p": 0.8, "type": "RandomPhotometricDistort"},
                         {"fill": 0, "type": "RandomZoomOut"},
-                        {"p": 0.8, "type": "RandomIoUCrop"},
+                        {"type": "RandomIoUCrop"},
```
Comment (aaronrmm) [inline rtdetr_pytorch/tests/unit/configs/test_rtdetr_r18vd_6x_coco_config.py:149]: - [ ] check if p is default

File: rtdetr_pytorch/README.md [hunk 5]
```diff
@@ -107,3 +107,10 @@ python tools/export_onnx.py -c configs/rtdetr/rtdetr_r18vd_6x_coco.yml -r path/t
 
 2. add `-t path/to/checkpoint` (optinal) to tuning rtdetr based on pretrained checkpoint. see [training script details](./tools/README.md).
 </details>
+
+
+<details open>
+<summary>list of wheel release</summary>
+
+rtrdetr-0.1.1-py3-none-any.whl - updated to use torchvision =< 0.16.0
+
```
Comment (aaronrmm) [inline rtdetr_pytorch/README.md:117]: - [ ] add 2.3 - fixed backward compatiblity

**Pull request merged**